tests/Bunny.test.ts

```diff
@@ -223,3 +223,74 @@ describe('BunnyCdnStream', () => {
     });
   });
 });
+
+describe(
+  'Collections API 2E2',
+  () => {
+    const stream = new BunnyCdnStream({ videoLibrary: process.env.BUNNY_VIDEO_LIBRARY || '123', apiKey: process.env.BUNNY_API_KEY || '123' });
+    let createdCollectionGUID: string;
+
+    test('Fetching a non-existing collection throws an error', async () => {
```

Comment:
Could you format these like the other tests, for example "GIVEN library w/o collection THEN throw error"

Reply:
Okay